packages/mdc-select/_mixins.scss

```diff
@@ -76,18 +82,26 @@
 }
 
 @mixin mdc-select-bottom-line-color_($color) {
-  .mdc-select__bottom-line {
-    @include mdc-select-bottom-line-fill-color($color);
+  .mdc-line-ripple {
+    @include mdc-line-ripple-color($color);
+  }
+
+  .mdc-select__native-control {
+    @include mdc-select-native-control-bottom-line-color_($color);
   }
 }
 
-@mixin mdc-select-focused-bottom-line-color_($color) {
-  @include mdc-select-focused-bottom-line_ {
-    @include mdc-select-bottom-line-fill-color($color);
+@mixin mdc-select-native-control-bottom-line-color_($color) {
+  border-bottom-color: $color;
+}
+
+@mixin mdc-select-focused-line-ripple-color_($color) {
+  @include mdc-select-focused-line-ripple_ {
+    @include mdc-line-ripple-color($color);
   }
 
-  .mdc-select__bottom-line::after {
-    @include mdc-select-bottom-line-fill-color($color);
+  .mdc-line-ripple::after {
```

Comment:
::after?? whats this for?

Reply:
Not sure what it was even doing before. Removing the rule.